tests/Functional/Fixtures/symfony/config/packages/seo_override.yaml

```diff
@@ -0,0 +1,26 @@
+seo_override:
+  fetchers:
+    - doctrine
+    -
+      type: in_memory
+      overrides:
+        domain_in_memory:
+          '/':
+            title: 'new title for homepage of domain_in_memory'
+        '':
+          '/error':
+            title: 'new title for error page'
+          '/admin':
+            title: 'new title for admin'
```

Comment:
Isn't the indent strange here? ('/' and '' does not looks like there are on the same level)

Reply:
Config didn't change according to https://github.com/jolicode/seo-override/pull/45/files/426eeb80b0a4f8cae81f500546cd0ba9949d47bc#diff-f07e70907118f2aff5532db7217d0b80999548fd2e0ab987f154250c5ccf084eL33-L58

Reply:
`'/'` is an override for the domain alias `domain_in_memory` only while `''` is the configuration for the default domain